### ISSUE TITLE
Possibility to have no more NVDA key defined

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#14527]
Note: issue similar to #14170.

### Steps to reproduce:
1. Start NVDA with a brand new empty config: `nvda -c %tmp%\nvda_folder_that_does_not_exist_yet`
2. In the welcome dialog, check the checkbox to use capslock as NVDA key
3. In the input gesture dialog, add a shortcut key to open keyboard parameters, which does not involves NVDA key, e.g. shift+space.
4. Open Notepad
5. Create an application profile for Notepad
6. Open keyboard parameters dialog
7. In the list to select NVDA modifier keys in keyboard params, uncheck Insert and Numpad Insert, then validate
8. Alt+Tab to an app other than Notepad, e.g. Windows Explorer
9. Open keyboard parameters dialog
10. In the list to select NVDA modifier keys in keyboard params, uncheck Insert and caps lock, then validate.
11. Alt+Tab to notepad and try to use caps lock, insert or numpad insert as NVDA key
12. Open keyboard parameters dialog
### Actual behavior:
* At step 11, there is no NVDA key at all.
* At step 12, we confirm that no key is configured as NVDA key

### Expected behavior:
In NVDA keyboard parameters dialog, when unchecking the 3 possible NVDA keys and validating, there is an error message indicating that at least one should be selected. Thus, there should not be any other mean to have them unchecked all three.

### NVDA logs, crash dumps and other attachments:
None
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2022.4
#### Windows version:
Windows 10 20H2 (AMD64) build 19042.2251
#### Name and version of other software in use when reproducing the issue:
None
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, already tested on NVDA 2022.3.x
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested, not related.
